book/src/gobject_signals.md

```diff
@@ -13,15 +13,16 @@ In our "Hello World" example we [connected](../docs/gtk4/trait.ButtonExt.html#ty
 {{#rustdoc_include ../listings/gobject_signals/1/main.rs:callback}}
 ```
 
-If we wanted to, we could have connected to it with the general (but much more verbose) `connect_local` method.
+If we wanted to, we could have 
+connected to it with the generic `connect_local` method.
```

Comment:
It would be nice to link to ObjectExt::connect_local here

Reply:
Good point!